### PR TITLE
Connect workflow

### PR DESCRIPTION
**Pull request by Rodri77**
Pending:

- Integrate popover filter that @snrondina is working on.
- Style tweaks as necessary.
- Test several screen sizes.

### CODE REVIEW

**Review by pellicceama — CHANGES_REQUESTED** [review]
Nice work! Please attach a video walkthrough with these for easier QA

Three requested changes
- Can we align search on the left as per figma?
- can we make connection card entire div pressable as opposed to just '+ add' 

https://www.loom.com/share/abb43d369105423baa6daf42635f844a

In smaller screens can we make the greenhouse (and other) modals show vertically centered?

image

Similar to larger screens

image

May be hard to do but worth having a view on this

**Comment by Rodri77** [comment]
> Nice work! Please attach a video walkthrough with these for easier QA
> 
> Three requested changes
> 
> * Can we align search on the left as per figma?
> * can we make connection card entire div pressable as opposed to just '+ add'
> 
> https://www.loom.com/share/abb43d369105423baa6daf42635f844a
> 
> In smaller screens can we make the greenhouse (and other) modals show vertically centered?
> 
> image
> Similar to larger screens
> 
> image
> May be hard to do but worth having a view on this

Resolved in the last couple commits

File: packages/engine-frontend/components/IntegrationSearch.tsx [hunk 1]
```diff
@@ -58,42 +69,46 @@ export function IntegrationSearch({
...
-                </Card>
-              )}
-            </WithConnectorConnect>
-          ))}
+        <div className="space-y-6 overflow-y-auto py-4">
+          {Object.entries(intsByCategory ?? {}).map(
+            ([category, categoryInts]) => (
+              <div key={category}>
+                <h3 className="mb-2 text-lg font-semibold">
+                  {category
```
Comment (pellicceama) [inline packages/engine-frontend/components/IntegrationSearch.tsx:77]: Can we make this that if the length of this is < 5 characters without spaces (I.e. HRIS) to put it in caps? That way we show ATS and not Ats.

**Review by pellicceama — CHANGES_REQUESTED** [review]
LGTM! small change

File: packages/engine-frontend/components/IntegrationSearch.tsx (hunk 1 above)
Comment (Rodri77) [reply packages/engine-frontend/components/IntegrationSearch.tsx]: Done in the last commit

**Pull request merged**